examples/dedicated_vmseries_and_autoscale/example.tfvars

```
@@ -0,0 +1,281 @@
+# --- GENERAL --- #
+location            = "North Europe"
+resource_group_name = "autoscale-common"
+name_prefix         = "example-"
+tags = {
+  "CreatedBy"   = "Palo Alto Networks"
+  "CreatedWith" = "Terraform"
+}
+enable_zones = false
+
+# --- VNET PART --- #
+vnets = {
+  "transit" = {
+    name          = "transit"
+    address_space = ["10.0.0.0/25"]
+    network_security_groups = {
+      "management" = {
+        name = "mgmt-nsg"
+        rules = {
+          vmseries_mgmt_allow_inbound = {
+            priority                   = 100
+            direction                  = "Inbound"
+            access                     = "Allow"
+            protocol                   = "Tcp"
+            source_address_prefixes    = ["134.238.135.137", "130.41.247.15"]
+            source_port_range          = "*"
+            destination_address_prefix = "10.0.0.0/28"
+            destination_port_ranges    = ["22", "443"]
+          }
+        }
+      }
+      "public" = {
+        name = "public-nsg"
+      }
+    }
+    route_tables = {
+      "management" = {
+        name = "mgmt-rt"
+        routes = {
+          "private_blackhole" = {
+            address_prefix = "10.0.0.16/28"
+            next_hop_type  = "None"
+          }
+          "public_blackhole" = {
+            address_prefix = "10.0.0.32/28"
+            next_hop_type  = "None"
+          }
```

What do you think about adding blackhole for `appgw` subnet ?

```suggestion
          }
          "appgw_blackhole" = {
            address_prefix = "10.0.0.48/28"
            next_hop_type  = "None"
          }
```